### ISSUE TITLE
Input field inserts but does not re-populate on re-render.

### ISSUE THREAD

**Issue by mgruen** [jshimko/meteor-autoform-selectize#1]
Thanks for publishing this package.

I apologize if this is an oversight on my part.   I am using Autoform 5.0.3. I have a helper with:

```
Template.registerHelper( "ops_skills" , function () { return { 
"Oracle EBS-GL": "Oracle EBS-GL",
"Oracle EBS-PO": "Oracle EBS-PO" 
}; } );
```

schema definition:
    skillTags:  {type: [String], optional:true,  maxCount:8, label:"Tags"  }

template tag:
        {{> afQuickField name="skillTags" label=false type="selectize" multiple=true options=ops_skills }}

in meteor mongo I get: 

```
"skillTags" : { "0" : "Oracle EBS-PO", "1" : "Oracle EBS-GL" }
```

but on re-loading the page nothing appears in the input field.  Is the value supposed to be a dict?  I was expecting from the schema def to see an array of strings?

Side effects I also experienced, which may be relevant:
- the field is not respecting maxCount.
- a nuanced error: if I select two items and save, I get {"0":"Item0", "1":"Item1"} .  If I immediately click on Item0 and delete it, then save, I end up with  {"0":"Item1", "1":"Item1"}.

Cheers,

**Comment by jshimko** [comment]
Since it's essentially a select element, you need to define the source of the options available to that input (just like you'd have to for a standard HTML select element with Autoform).

So, something like this...
```
{
    type: [String],
    optional: true,
    autoform: {
      editable: true,
      options: function () {
        return Meteor.users.find().map(function (user) {
          return {
            value: user._id,
            label: getDisplayName(user)
          }
        });
      }
    }
  }
}

**Comment by jshimko** [comment]
Since it's essentially a select element, you need to define the source of the options available to that input (just like you'd have to for a standard HTML select element with Autoform).

So, something like this...

``` js
// skillTags Schema
skillTags: {
    type: [String],
    optional: true,
    autoform: {
      options: function () {
        return Tags.find().map(function (tag) {
          return {
            value: tag._id,
            label: tag.name
          }
        });
      }
    }
  }
}
```

**Comment by jshimko** [comment]
I guess I should probably update the README with some examples.  :)

I'll take care of that tonight when I have a little more time available.

**Comment by mgruen** [comment]
Thanks very much for the response - I did try your example above - or rather, to be precise, an array of statics, thus:

```
      options: function () {
        return [
          {value: "2013", label: "2013"},
          {value: "2014", label: "2014"},
          {value: "2015", label: "2015"}     ]  }
```

But I keep getting the pattern {"0":"2014", "1":"2015"} as the value in the mongodb document, and then no values showing in the input field on re-rendering.  While testing, I installed jeremy:autoform-selectize on a version of the app with old values in the db - I fired it up and the values rendered just fine - I then looked in the document and the key had a simple array of strings ( ["2013", "2014"] ).

I'm a beginner in this space so I suspect I may be missing something obvious, perhaps elsewhere in the stack.  
Regarding the pattern of the options for the select element, just one comment that may be helpful: the autoform docs suggest a couple of different ways to define options for array fields, and many users may be using the shortened "alternative syntax", so it might be worth a mention as well if not compatible:

https://github.com/aldeed/meteor-autoform#use-allowed-values-array-from-the-schema-as-both-the-label-and-the-value
